idd/Energy+.idd.in

```diff
@@ -42751,6 +42773,93 @@ Coil:Heating:DX:VariableRefrigerantFlow,
+  A2,  \field Availability Schedule Name
+       \type object-list
+       \object-list ScheduleNames
+       \note Enter the name of a schedule that defines the availability of the coil
+       \note Schedule values of 0 denote the unit is off. All other values denote the unit is available
+       \note If this field is left blank, the unit is available the entire simulation
+  A3,  \field Coil Air Inlet Node
+       \required-field
+       \type node
+       \note the inlet node to the coil
+  A4,  \field Coil Air Outlet Node
+       \required-field
+       \type node
+       \note the outlet node to the coil
+  N1,  \field Rated Total Cooling Capacity
+       \required-field
+       \type real
+       \units W
+       \minimum> 0.0
+       \note Supply air fan heat is not included
+  N2,  \field Rated Sensible Heat Ratio
+       \required-field
+       \type real
+       \minimum> 0.0
+       \note Supply air fan heat is not included
+  N3,  \field Indoor Unit Reference Superheating 
+       \required-field
+       \type real
+       \units deltaC 
+  A5,  \field Indoor Unit Evaporating Temperature Function of Superheating Curve Name    
+       \required-field   
+       \type object-list
+       \object-list QuadraticCurves
+       \object-list UniVariateTables
+  A6;  \field Name of Water Storage Tank for Condensate Collection
+       \type object-list
+       \object-list WaterStorageTankNames
+ 
+Coil:Heating:DX:VariableRefrigerantFlow:FluidTemperatureControl, 
+       \memo This object models the indoor unit coil of the Variable Refrigerant Flow system (heating mode)
+       \memo Used with ZoneHVAC:TerminalUnit:VariableRefrigerantFlow. 
+       \memo Outdoor unit is modeled separately, see AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl.
+       \min-fields 6
+  A1,  \field Name
+       \required-field
+       \type alpha
+       \reference HeatingCoilsDXVarRefrigFlowFluidTemperatureControl
+  A2,  \field Availability Schedule
+       \type object-list
+       \object-list ScheduleNames
+       \note Enter the name of a schedule that defines the availability of the coil
+       \note Schedule values of 0 denote the unit is off. All other values denote the unit is available
+       \note If this field is left blank, the unit is available the entire simulation
+  A3,  \field Coil Air Inlet Node
+       \required-field
+       \type node
+       \note the inlet node to the coil
+  A4,  \field Coil Air Outlet Node
+       \required-field
+       \type node
+       \note the outlet node to the coil
+  N1,  \field Rated Total Heating Capacity
+       \required-field
+       \type real
+       \units W
+       \minimum> 0.0
+       \autosizable
+       \note Supply air fan heat is not included
+  N2,  \field Indoor Unit Reference Subcooling 
+       \required-field
+       \type real 
+       \units deltaC
+  A5;  \field Indoor Unit Condensing Temperature Function of Subcooling Curve Name    
```

Comment:
Have you tried using autosize for Rated Total Heating Capacity? Is it possible to autosize this input? Can you add a default for Indoor Unit Reference Subcooling since this value is typically around 5C (check this)? If you do add a default, remove the \required-field tag.